src/main/resources/electronic-data-mapping.json

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
Just a suggestion, if you think it makes sense, to put the mappings in alphabetical order, as it's easier to organize and to find a mapping.